src/pages/HomePage.js

```diff
@@ -12,11 +12,19 @@ import { StyledBox } from "../components/commonStyles";
 
 const HomePage = () => {
   const { state } = useContext(GlobalContext);
-  const { categories } = state;
+  const { categories} = state;
+
   let navigate = useNavigate();
+
   const handleClick = (id) => {
     navigate(`/item/${id}`);
   };
+
+  // Now that categories is an object, we get an array of all the category ids
+  const categoryIds = Object.keys(categories);
+  // then create an array of the category objects to pass into the components that were expecting arrays
+  const arrayOfCategories = categoryIds.map(id => categories[id])
```

Comment:
here's is how i'm using the `categories` object to create an array to of category objects to pass into the `ListOfItems` so that none of that code had to change

Reply:
pro 💪